apps/pie-docs/src/components/card-container/card-container.md

```diff
@@ -0,0 +1,71 @@
+---
+eleventyNavigation:
+    key: 'Card Container'
+    parent: Components
+---
+
+## Overview
+
+The purpose of cards is to provide a cohesive and structured way of displaying content. Cards are interactive, allowing users to click or tap for more details or to perform specific actions related to the displayed item.
+
+Cards are designed to present concise and focused information organised in a visually appealing manner. They are commonly used to showcase individual items, such as products or articles, and are usually arranged in a grid or stacked layout.
+
+
+{% contentPageImage {
+    src:"../../assets/img/components/card-container/overview.svg",
+    alt: "A card container with text and an icon inside of it."
+} %}
+
+---
+
+## Resources
+
+{% notification {
+  type: "warning",
+  message: "We’re currently working on updating our Card Container documentation, please see the resources below."
+} %}
+
+{% resourceTable {
+    rows: [
+        {
+            resource: resourceTypes.COMPONENT,
+            link: "https://www.figma.com/file/R2rBfzJP0hG0MZorq6FLZ1/%5BCore%5D-Components-%E2%9A%AA%EF%B8%8F-%5BPIE-2.0%5D?type=design&node-id=742-326&mode=design",
+            status: statusTypes.AVAILABLE
+        },
+        {
+            resource: resourceTypes.DOCUMENTATION,
+            link: "https://www.figma.com/file/j1YKygEyhqZ6zKVxcHapn5/%5BCore%5D-Component-Documentation-%E2%9A%AA%EF%B8%8F-%5BPIE-2.0%5D?type=design&node-id=742-326&mode=design",
+            status: statusTypes.AVAILABLE
+        },
+        {
+            resource: resourceTypes.WEB_COMPONENTS,
+            status: statusTypes.PLANNED
```

Comment:
The pie-card-container is currently on the webc storybook site; at which point do we move this to a different status? (If this can be updated now I'm happy for this to be done in a following PR)

Reply:
good question, I'm not sure when it changes but it probably makes sense for it to have a PRE_RELEASE status instead since it's on the storybook site. I'll ping design to double check as I got this info from their google sheet 😄 if it's cool to change, I'll update in the next PR 👍 thank you!